src/pkg/k8s/configmap.go

```diff
@@ -0,0 +1,59 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ReplaceConfigmap deletes and recreates a configmap
```

Comment:
All of these comments will flag on godot.

Reply:
punting until next time, because that's a lot-o-dots